src/screens/FilterScreen.js

```diff
@@ -242,14 +242,14 @@ class FilterScreen extends Component {
                     renderContent={this.renderContent}
                 />
 
-                <Accordion
-                    underlayColor={'#00000000'}
-                    initiallyActiveSection={collapsedState}
-                    style={accordionFilterStyle}
-                    sections={FOREST}
-                    renderHeader={this.renderHeader}
-                    renderContent={this.renderContent}
-                />
+                {/*<Accordion*/}
+                    {/*underlayColor={'#00000000'}*/}
+                    {/*initiallyActiveSection={collapsedState}*/}
+                    {/*style={accordionFilterStyle}*/}
+                    {/*sections={FOREST}*/}
+                    {/*renderHeader={this.renderHeader}*/}
+                    {/*renderContent={this.renderContent}*/}
+                {/*/>*/}
```

Comment:
Commented out Forest filter until we have more forests in the app.  Didn't remove it since we'll probably want to add it back eventually.